src/exporto0o.ts

```diff
@@ -141,23 +129,33 @@ export async function exportToOo(
     variables.outputFileName = variables.outputFileFullName.substring(0, variables.outputFileFullName.lastIndexOf('.'));
   }
 
+  const env = Object.fromEntries(Object.entries(setting.env ?? {}).map(([n, v]) => [n, renderTemplate(v, variables)]));
+
   // show progress
   progress.setMessage(lang.preparing(outputFileFullName));
   beforeExport && beforeExport();
   progress.show();
 
   const pandocPath = getPlatformValue(globalSetting.pandocPath) ?? 'pandoc';
 
-  const cmdTpl =
+  const templateOption = templateOptions.find((option) => option.value === exportTemplate); // Use exportTemplate parameter
+  const templatePath = templateOption ? templateOption.path : '';
+  let cmdTpl =
     setting.type === 'pandoc'
-      ? `${pandocPath} ${setting.arguments ?? ''} ${setting.customArguments ?? ''} "${currentPath}"`
+      ? `${pandocPath} ${setting.arguments ?? ''} ${setting.customArguments ?? ''}`
       : setting.command;
-  const cmd = generateCommand(cmdTpl, variables);
+  // if template is selected, append to command
+  if (exportTemplate !== 'none') {
```

Comment:
This is unnecessary.  The template configuration already does this automatically.


You revert my changes.